src/index.js

```diff
@@ -0,0 +1,93 @@
+import './style.css';
+
+let tasks = [];
```

Comment:
- [Optional] Consider adding some hardcoded todos in your list so that you can see what the styling will look like.